dotCMS/src/main/java/com/dotmarketing/db/DbConnectionFactory.java

```diff
@@ -438,6 +438,38 @@ public static boolean isDBTrue(String value) {
 
     }
 
+    /**
+     * Return true if the String represent one on the false
+     * boolean value for the current database
+     * @param value String representation of the boolean
+     * @return true is the string represents a DB false value
+     */
+    public static boolean isDBFalse(final String value) {
+        String x = getDBType();
+
+        if (MYSQL.equals(x)) {
+            return value.trim().equals("0") || value.trim().equals("false");
+        } else if (POSTGRESQL.equals(x) || H2.equals(x)) {
+            return value.trim().equals("f") || value.trim().equals("false");
+        } else if (MSSQL.equals(x)) {
+            return value.trim().equals("0");
+        } else if (ORACLE.equals(x)) {
+            return value.trim().equals("0");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "0" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)